test/performance/run-test.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+if test -f .env; then
+  source .env
+fi
+
+CL2_HOME_DIR=${CL2_HOME_DIR:=/Users/johny/perf-tests/clusterloader2} 
+CL2_BINARY_NAME=${CL2_BINARY_NAME:=clusterloader}
+
+USE_KUEUE=${USE_KUEUE:=true}
+
+DEFAULT_EXPERIMENTS=(
+    "10 2 0 2s 3m 100 100Gi"
+)
+
+EXPERIMENTS=("${EXPERIMENTS[@]:=${DEFAULT_EXPERIMENTS[@]}}")
+
+KUBECONFIG=${KUBECONFIG:="$HOME/.kube/config"}
+export KUBECONFIG
+
+PROVIDER=${PROVIDER:=gke}
+
+export EXEC_DEPLOYMENT_YAML="$CL2_HOME_DIR/pkg/execservice/manifest/exec_deployment.yaml"
+
+cp -r "$CL2_HOME_DIR/pkg/prometheus/manifests/" tmp_manifests
+trap 'rm -r tmp_manifests' EXIT
+PROMETHEUS_MANIFEST_PATH=$(pwd)/tmp_manifests
+export PROMETHEUS_MANIFEST_PATH
+
+now=$(date +%Y-%m-%d-%H.%M.%S)
+
+if [[ "$USE_KUEUE" == true ]]; then
+    export CL2_USE_KUEUE=true
+    kubectl kustomize ../../config/prometheus | yq -s '.kind' -o yaml
```

Comment:
maybe `kubectl kustomize -o` would allow you to write directly into `$PROMETHEUS_MANIFEST_PATH`?

Reply:
Kustomize places all Kubernetes object manifests (`role`, `rolebinding` and `servicemonitor`) in the same file, however, Clusterloader expects, that there is one manifest per file, otherwise it does not create all the objects from the file. That's why I added `yq` expression, which splits produced manifest into 3 files.

Reply:
sounds good. Please add a comment in the code for what you just said.